test/units/testsuite-17.11.sh

```diff
@@ -237,9 +240,14 @@ test_syntax_error 'LABEL=="b"' 'Invalid operator for LABEL.'
 test_syntax_error 'LABEL="b"' 'LABEL="b" is unused.'
 test_syntax_error 'a="b"' "Invalid key 'a'"
 test_syntax_error 'KERNEL=="", KERNEL=="?*", NAME="a"' 'conflicting match expressions, the line takes no effect'
+test_syntax_error 'KERNEL!="", KERNEL=="?*", KERNEL=="", NAME="a"' 'conflicting match expressions, the line takes no effect'
```

Comment:
It could be moved, but the idea behind this particular test is to check that there are no diagnostics about duplicate tokens in the line where conflicting tokens were found, because conflicting tokens invalidate the whole rule.